notebooks/daniel_notebook.ipynb

```diff
@@ -1,191 +1,34 @@
 {
```

Comment:
<p>you can use <code>\times</code>  in the latex</p><p>$$\times$$</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cooperrc/uconn_numpy_tutorials/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>